### PR TITLE
Added-masonary-layout-files

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

**Review by mmusk — APPROVED** [review]

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+body {
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css:1]:
```suggestion
* {
margin : 0;
padding: 0;
box-sizing: border-box;
}
body {
```

File: tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  margin: 0;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css:2]: Remove this

File: tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css (hunk 1 above)
Comment (Abbas4tech) [reply tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css:1]: Added CSS Reset ! @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/08-Responsive-masonary-layout-css-grid/style.css]: Done @Abbas4tech

**Review by Abbas4tech — APPROVED** [review]

**Pull request merged**